tests/cupyx_tests/jit_tests/test_raw.py

```diff
@@ -69,6 +69,7 @@ def f(x, y):
         f((1,), (1,), (x, y))
         assert bool((x == y).all())
 
+    #@unittest.skipIf(cupy.cuda.runtime.is_hip, 'segfault?')
     def test_syncthreads(self):
```

Comment:
This test passed on Radeon VII pro + ROCm 4.0.0, but segfaults on Radeon VII + ROCm 3.5.0. I suspect it's due to either resource allocation, which could be constrained by `__launch_bounds__`, or bugs that are fixed in recent ROCm, so let's just keep a note and not touch them in this PR.